tutor/src/screens/new-course/sociology-3e-nudge.tsx

```diff
@@ -28,7 +28,7 @@ const Sociology3eNudge = ({ ux }: { ux: any }) => {
     if (ux.displaySoc3eAvailableNudge) {
         return (
             <Text>
-                Introduction to Sociology 2e is available until Summer 2022. Upgrade to <cite>Introduction to Sociology 3e</cite>.
+                Introduction to Sociology 2e is available until Summer 2022. <a href={`/new-course/offering/${ux.offerings.sociology3e.array[0].id}`}>Upgrade to <cite>Introduction to Sociology 3e</cite>.</a>
             </Text>
```

Comment:
Hehe, so while you can use `a` tags and it'll work that's not the optimal way to change pages.  The browser will do a whole page reload when it follows them. It's better to use the `Link` component like is used here https://github.com/openstax/tutor-js/blob/main/tutor/src/components/my-courses/create-a-course.jsx#L11 That will just push a new entry into the browser's `history` and react-router will update only the needed components.